trio/_core/_run.py

```diff
@@ -845,6 +914,15 @@ def started(self, value=None):
         self._old_nursery._check_nursery_closed()
 
 
+class _NurseryStartFunc(Protocol[StatusT_co]):
+    """Type of functions passed to `nursery.start() <trio.Nursery.start>`."""
+
+    def __call__(
+        self, *args: Any, task_status: TaskStatus[StatusT_co]
```

Comment:
Have you made sure the protocol matches a couple representative functions? I'm a bit scared the `*args` might require all callables passed to have `*args` of their own.

Reply:
It might actually. So then even `TypeVarTuple` won't help, we'll need a heap of protocols or something.

Reply:
Nah I've tried out TypeVarTuple, the difference here is that *args: object is an infinite amount vs *args: Unpack[Ts] is a finite (probably) amount)